packages/synapse-react-client/src/lib/containers/MarkdownPopover.tsx

```diff
@@ -1,124 +1,106 @@
+      <Typography variant="body1" marginBottom={2}>
+        <MarkdownSynapse {...contentProps} />
+      </Typography>
+      <Box sx={buttonBoxSx}>
+        {actionButton && (
+          <Button
+            color={actionButton.color || 'primary'}
+            variant={actionButton.variant || 'contained'}
+            onClick={() => {
+              actionButton.onClick()
+              if (actionButton.closePopoverOnClick) {
+                setShow(false)
+              }
+            }}
+          >
+            {actionButton.content}
+          </Button>
+        )}
+        {showCloseButton && (
+          <Button variant="outlined" onClick={() => setShow(false)}>
+            Close
+          </Button>
+        )}
+      </Box>
+    </Box>
   )
-  const [arrowElement, setArrowElement] = useState<HTMLDivElement | null>(null)
 
-  const { styles, attributes } = usePopper(referenceElement, popperElement, {
-    placement: placement,
-    strategy: strategy,
-    modifiers: [
-      { name: 'arrow', options: { element: arrowElement } },
-      {
-        name: 'offset',
-        options: {
-          offset: [0, 10],
+  return (
+    <LightTooltip
+      title={content}
+      placement={placement}
+      onClick={() => setShow(!show)}
+      open={show}
+      sx={{
+        ...sx,
+        [`& .${tooltipClasses.tooltip}`]: {
+          maxWidth: { maxWidth },
+          minWidth: { minWidth },
         },
-      },
-    ],
-  })
-
-  const popover = (
-    <CSSTransition
-      in={show}
-      timeout={200}
-      classNames="fade-in-out"
-      mountOnEnter
+      }}
     >
-      <div
-        className={`Tooltip SRC-popover bootstrap-4-backport`}
-        style={{ ...style, ...styles.popper }}
-        ref={setPopperElement}
-        {...attributes.popper}
-      >
-        <Typography variant="body1">
-          <MarkdownSynapse {...contentProps} />
-        </Typography>
-        <div className="TooltipButtonContainer">
-          {actionButton && (
-            <Button
-              color="primary"
-              variant="contained"
-              onClick={() => {
-                actionButton.onClick()
-                if (actionButton.closePopoverOnClick) {
-                  setShow(false)
-                }
-              }}
-            >
-              {actionButton.content}
-            </Button>
-          )}
-          {showCloseButton && (
-            <Button variant="outlined" onClick={() => setShow(false)}>
-              Close
-            </Button>
-          )}
-        </div>
-        <div
-          className="arrow"
-          data-popper-arrow
-          ref={setArrowElement}
-          style={styles.arrow}
-        />
-      </div>
-    </CSSTransition>
-  )
-
-  return (
-    <>
-      <span
+      <Box
         role="button"
-        className="PopoverContainer bootstrap-4-backport"
-        onClick={() => setShow(val => !val)}
-        ref={setReferenceElement}
+        className={'PopoverContainer'}
```

Comment:
Class is used for positioning the HelpPopover info icon in DownloadCartPage --

main | feature
:---:|:---:
![main_HelpPopover_DownloadCartPage](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/f691fea6-2128-4fe8-b846-db8df06fceed) | ![feature_HelpPopover_DownloadCartPage](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/291c5c55-06e2-4727-855d-f2c7e96cbd7c)